### ISSUE TITLE
A minimalist comic about the evolution of reasoning and heated conversations

### ISSUE THREAD

**Issue by janzeteachesit** [janzeteachesit/100-days-of-writing#1060]
A minimalist comic about the evolution of reasoning and heated conversations http://ift.tt/2w90QSg GRIST4&nbsp; Writing&nbsp; A minimalist comic about the evolution of reasoning and heated conversations Starring a sphere, a cube, and some words in&nbsp;between. Buster BensonJul 4 &#128576;&#128173; Thinking is hard. Platform product lead at @SlackHQ. Also helping a bit with @coachdotme, @NaNoWriMo, @750words. Previously @Twitter and @Amazon. First, some back&nbsp;story About a year ago, I published the Cognitive Bias Cheat Sheet, which attempted to synthesize all 200+ cognitive biases on Wikipedia&rsquo;s list of cognitive biases page into a more organized framework based on the jobs they were hired to do in our brains. A few months later, I tried to simplify it again into 4 basic qualities of the universe that, though impossible to solve entirely, our brains had evolved to mitigate in clever ways. One side effect of those helpful shortcuts was that we tended to systematically get some things wrong. Together, these articles are approaching a million reads, and it continues to attract new readers. If you search for cognitive biases on Google, the first article is consistently in the top 3 results, and the cool poster that John Manoogian III created to accompany the post appears above the results in Google&rsquo;s featured snippet slot. I feel lucky to have hit on a topic people are interested in. The problem Despite that success, something has continued to nag me. Part of it is that even though I now see bias everywhere (thanks, familiarity effect!), I feel like it&rsquo;s still too difficult to apply this information to real life. I frequently ask myself: &ldquo;What measures am I taking to combat my own&nbsp;biases?&rdquo; My brain can confabulate a few things, like &ldquo;you&rsquo;re writing a book about this stuff!&rdquo; and &ldquo;you&rsquo;re probably not as bad as most people!&rdquo; but they are excuses rather than real answers. A broken part of our&nbsp;lives What&rsquo;s something we as humans do all the time, and that feels suuuuper broken today? That&rsquo;s easy: arguing! How many conversations have you been in recently where you&rsquo;ve attempted to discuss a topic (either online or in person) and have watched it devolve into personal insults, leaving you feeling more disconnected and frustrated with the people you&rsquo;re talking with than you did when you started? How much better would life be if we somehow &#10024; magically &#10024; improved our ability to have heated conversations with people about the topics that were important to us? Could we eventually tiptoe out of our filter bubbles where, in full honestly, we just practice arguing with imaginary people that aren&rsquo;t present, as a way to feel right about something and burn through those emotions in a safe space? (Hope that&rsquo;s not too harsh&hellip; I do it too.) Could we actually move from the frustrating &ldquo;Gah! These people don&rsquo;t make any sense! They&rsquo;re idiots! Smash everything!&rdquo; to a place where we are able to (probably slowly at first) learn how people different from us think, find out why, share our own perspectives, listen, get heard, and find common ground amongst the most controversial topics? It has felt like it would take a miracle. But&hellip; maybe it wouldn&rsquo;t? I also began asking friends about the kinds of arguments they were getting in. With people on Facebook, with relatives, etc. People have been unfriending and blocking and ending long friendships left and right! Whatever is happening in the world, and our country, is also happening in our relationships. I also became curious about those arguments that last months and years, the kind that keep bubbling up in weird ways over and over again, but never get resolved. Even healthy relationships have them. As I asked around, the variety of long-lived arguments was impressive: from whether or not water left out a few days was still safe to drink, to whether or not one movie/album/show is objectively better than another, to the much more serious questions around money, cleanliness, and respect. I also became curious about arguments that pop up at work, when we&rsquo;ve got the added variables of an org chart, and departments, and job descriptions, and deadlines. The same kinds of patterns I saw online, and amongst friends, came up in this context too! Finally, I asked myself: are biases involved in how we argue and impact how successful they are? Duh, of course! In fact, there&rsquo;s a fascinating line of cognitive psychology that was kicked off by a paper titled, The Argumentative Theory of Reasoning. It proposes that our capacity to reason evolved BECAUSE it was evolutionarily advantageous to be a good arguer, and to win arguments. We reason in order to have better arguments, not the other way around. Are biases may be intentional features of our brains rather than bugs&hellip; they help us construct better arguments! And that&rsquo;s how this topic elevated to become a light obsession for me. A shift of attention In short&hellip; I have a new plan to shift my focus off of biases explicitly, and turn it more to the social practice of arguing (which our biases are heavily implicated in). Because: Arguments are present in every part of our lives, and they impact our quality of life. They aren&rsquo;t going away anytime soon. It seems like a worthwhile investment to try to get better at them. Whatever better means to each of us. Getting even a little better at using this deeply programmed social ritual could make each of our lives significantly more meaningful and generally pleasant. The upside is enormous. It also seems like a huge task, but that&rsquo;s okay. Steps to get&nbsp;there Immerse myself in the general problem space of reason, conversation, and arguing. Learn as much as I can. Dive deep. (This is what I&rsquo;ve been doing the last 10 days.) Share what I&rsquo;ve learned with others, experiment with different strategies for exploring this topic with different audiences (webcomics, maybe?), ask for feedback. (That&rsquo;s what this post is about.) Invite people to have conversations about specific areas of disagreement, dissect them, practice having them, test different approaches to see how they work. (I&rsquo;ve started doing this with a few trusted friends and will expand the circle once I need more of a challenge&hellip; here&rsquo;s an example. If you&rsquo;re interested, follow this Medium publication.) Rinse and repeat. Build, measure, learn&hellip; argue. Story time. Here&rsquo;s my silly (but at times serious) story that attempts to share nascent thoughts on the problem-universe of reason, arguing, and heated interactions. I hope you like it. Please let me know either way! The story of Sphere and&nbsp;Cube This story begins near the beginning of the universe, with the simplest of merry protagonists: Sphere. One day something novel happened. She hit something. What was it? She couldn&rsquo;t tell. It was new. She felt a rumbling in her ever-elusive subconscious mental fog, and it coalesced into a burp with a newly conscious presence: a question! (An aside: I&rsquo;m using fiftythree&rsquo;s Paper app to make these because they have a really nice way to make consistent shapes like squares and circles. I&rsquo;m not going to lie&hellip; that had a large role in helping me figure out who the characters in this story would&nbsp;be.) &#10024; First Conscious Thought achievement unlocked! &#10024; Like I said, the passage of time is a bit vague. Not much ever happened, so it wasn&rsquo;t necessary to check the calendar often. Things like this happened pretty much every couple seconds, all over the place. But one day, Sphere opened her eyes (how long had they been there?), thus flooding her brain with an dazzling new firehose of information to consider in addition to the subconscious burps. A new thought formed: Having eyes helped this lightbulb go off, for sure. And soon everyone had them because they got quite&nbsp;trendy. &#9989; Vision skill added! &#9989; The previously impenetrable mystery of what they were running into and what they were falling off found quick answers. &#128064; Cube discovered! &#128064; &#128064; Cliffs discovered! &#128064; &#10024; First Question Answered achievement unlocked! &#10024; But new problems emerged. &#9760;&#65039; Sphere fell to her death. &#9760; Over many generations, new defenses evolved (we&rsquo;ll skip a bunch that didn&rsquo;t end up working very well, or were downright useless), and things began to take on a distinctly &ldquo;pleasant&rdquo; or &ldquo;unpleasant&rdquo; flavor in her mind. This helped guide behavior out of the realm of pure chaos and randomness. &#10024; Anger achievement unlocked! &#10024; &#9989; Self Defense skill added! &#9989; &#10024; Yay achievement unlocked! &#10024; &#9989; Symbolic Language skill added! &#9989; This was going to be handy. Words helped Sphere represent her intentions without having to expend the energy required to light herself on fire. This saved time, energy, and ultimately lead to having more leisure time for pursuits other than clawing for basic survival. It also had a side benefit of giving Cube a better idea of Sphere&rsquo;s own thoughts, allowing him to act accordingly before being singed. &#10024; Shared Understanding achievement unlocked! &#10024; Sphere learned that other creatures, like Cube, also have agency. Sphere can even sort of guess what&rsquo;s going on in their own heads. It&rsquo;s far from perfect. One is bound to get a few things wrong here and there. &#128064; Large Cube discovered! &#128064; Like this slight oversight. Error noted. &#9760;&#65039; Sphere is burned to death. &#9760; Maybe a bit too late. Luckily, look who we have observing from a safe distance: &#10024; Speechless Horror achievement unlocked! &#10024; This is when Sphere&rsquo;s language and shared understanding skills go into overdrive, trying to make sense of the scene she has just witnessed. She knows that Cube and Large Cube both have their own idiosyncratic thoughts, feelings, and needs, but she doesn&rsquo;t have enough details to construct a full story about the motivations behind their actions. It just doesn&rsquo;t make sense. Which, in most brains, translates directly to bad. &#128064; Cognitive Dissonance discovered! &#128064; In situations like this, Sphere has two options. Either: There&rsquo;s something she misunderstands about Cube, and she should look into it. It&rsquo;s clear that Cubes are monsters. That&rsquo;s an easy call. In fact, it supports her previous hunch. She uses this new evidence to construct a more detailed story about Cube&rsquo;s motivations. Cubes are murderous barbarians. Cubes are evil. They must be destroyed. &#10024; Confirmation Bias achievement unlocked! &#10024; &#10024; First Goal achievement unlocked! &#10024; Another side effect of categorizing the contents of the world (good things, bad things, scary things, happy things, etc) is that Sphere&rsquo;s brain sometimes makes the wrong connection and misinterprets some information. Even though the mistake is caught shortly after being made, the fast part of her brain (system 1) has already made the neural connection (it&rsquo;s really fast). No biggie, though, because if it turns out to be wrong the connection will fade over time. &#9989; Implicit Associations skill added! &#9989; Stories, motives, and associations tangle up in a giant knot in her mind. She loops over the same connections and conclusions over and over again, re-igniting the emotions each time. This, in effect, insures that the neural pathways in her head are wired in a more permanent way. What fires together wires together. Once burned in, these thoughts linger longer in her nerves, blood, and thoughts. Exhausted by it all, she heads home. Home is safe. In time, her mind and body settle down enough to transition into a half-sleep. Her brain transitions into dream maintenance mode, where the night shift workers clean up and organize the mess left by the day&rsquo;s chaotic thoughts. Goodnight! &#128164; &#127942; Level 1&#65039;&#8419; complete! &#128079; &#128079; &#128079; &#128275; Level 2&#65039;&#8419; unlocked! &#128079; &#128079; &#128079; Level 2 Some time passes. More than a night, less than an epoch. Brains don&rsquo;t just sit still in the dark, unchanging. They&rsquo;re more like a jar of fermenting sauerkraut (that seems more accurate than wine, for some reason). Left alone, some flavors deepen, and others fade away. It&rsquo;s not an entirely predictable process, so you should check in on it every once in a while to see what&rsquo;s happening. Wipe off any fuzzy mold that you find growing there. And while the universe is extremely large, it also has a funny way of bringing people back to the same few themes over and over again. Here&rsquo;s Cube, knocking on Sphere&rsquo;s door. &#10024; Winning Argument achievement unlocked! &#10024; This is a big deal! Let&rsquo;s pause here a moment to add some context. This is a story about the evolution of reason and heated conversations. While it hasn&rsquo;t been explicitly called out, you can look back and see a number of different ways Sphere and Cube have learned to communicate, interpret what&rsquo;s going on, and resolve disputes. To generalize, there are: 3 different ways to interact with&nbsp;others 1. Brute Force: The first form of communication and debate was simply to force your will upon others and the world. Push them off a cliff. No need to ask first, because the only opinion that matters is your own. 2. Bargaining: This is a very broad category that includes everything from taunting physically and/or verbally (pushing, burning, and other tools of intimidation), to using punishment and rewards to come to a &ldquo;deal&rdquo;, to trading things of equal value that both sides agree to. 3. Reasoning: This is where Sphere and Cube find themselves now, if only in nascent form. It&rsquo;s about more than just an equal balance of known goods. There&rsquo;s a 3rd new element in play: reasons. Reasons are weird and a little magical. There&rsquo;s a sense, with Sphere, that no amount of money, or threats, or deal-making would get her to help Cube with whatever he&rsquo;s about to ask for. The reason: Cubes are murderers. The only way to resolve a dispute via reasoning is to provide even better reasons to do so. Cube sits with this new information. To his credit, he feels bad. (The brown color he turns is my not-so-subtle way of hinting that he feels like shit. I&rsquo;m still new at this, be easy.) Another important thing to call out about reason-based arguments, is reasons have intrinsic value on their own. Unlike in bargaining where the valuable goods are exchanged only once a deal has been made, in reason-based arguments, valuable goods are exchanged during the conversation. Here&rsquo;s why reasons are valuable on their own: &#128273; A reason is a glimpse into another person&rsquo;s thinking, and this can be used to 1) better understand that person&rsquo;s current perspective and 2) maybe even predict their future thoughts. Cube came to ask for Sphere&rsquo;s help. Cube didn&rsquo;t get that, but he did get a better understanding of her perspective, which is the key to getting her help. In a universe where many disputes will never be resolved, the strategy that gets some value out of those unresolved disputes will accumulate more value over time. If Cube had only been using brute force or bargaining, he would have left empty-handed. But here on both counts he did not. And now that Cube understands Sphere a bit better (maybe even to the point of agreeing with her about a few things), he&rsquo;s able to craft a better reason for Sphere to help him. This drawing is not the best, sorry. But Sphere has left her house with the door open, and sees some markings in the ground that encourage her, for the first time, to step off the 2-dimensional plane that this comic has so far operated&nbsp;within. The value of reasons goes both ways, too. Sphere, watching Cube from behind the door, also learned something about Cube. He now seems a bit different from the other Cubes (Large and Regular), at least, according to the story she had written about them. In her story, Cube should have attempted to break into her house, but he didn&rsquo;t. Now this strange message on the ground. Was her story about them incomplete? She couldn&rsquo;t help but be a little curious, and follow the arrow. (Hmm&hellip; so maybe that&rsquo;s what was growing in her brain sauerkraut&hellip;) &#10024; Curiosity achievement unlocked! &#10024; In moments of surprise, Sphere often reverts back to her earliest, protoself&#8202;&mdash;&#8202;that blind, undirected, rolling fog of subconscious energy, and her first burp of purest consciousness: Huh? What is this giant thing, balanced on a point? Sphere and Cube are starting to look like something approaching&hellip; acquaintances? They obviously aren&rsquo;t anything close to that, given things. To that point, though, here&rsquo;s another reason why reasoning has evolved to surpass the other methods: a side effect of using reasons to settle disputes, in addition to the fact that it gives both sides a glimpse into the other&rsquo;s mind, is that these glimpses makes others feel more predictable. You learn a tiny bit about how they think! You can use that to update your story about them. Assuming the reason provided passes your smell test, you can even begin to trust them more based on this glimpse. Best part about this is that it can happen even before they have given any outward evidence that would earn that added trust. Because it just makes sense. &#9989; Story Updating skill added! &#9989; 2 kinds of trust, which are acquired at different speeds Reason-based trust: Reasons provide glimpses into other minds, which make the other minds more predictable. This predictability generates an immediate trust boost within the domain that the reason applied. Experience-based trust: Earned by making promises and keeping promises repeatedly. This process essentially reverse-engineers the same predictability from above by triangulating on the way other minds work. Because it&rsquo;s unguided, it takes much longer to get there. &#9989; Reason-Based Trust skill added! &#9989; Trust, however, does not make you a better stair-climber, especially if you are a sphere. This took a lot longer than you&rsquo;d think, but they eventually made it up. &#128640; Congrats, Sphere! You&rsquo;ve reached your daily goal of 10,000 steps! &#128640; What was up there? What did they find? Again my drawing might be a bit difficult to decipher, but imagine a ledge high up on a hill, where Sphere and Cube are walking up to the edge of. And this giant dome is reaching up from the ground. So they couldn&rsquo;t walk over to it, they&rsquo;d fall off the ledge. I probably need to do this one&nbsp;again. A giant spherical dome reaches up from the ground, and like before Sphere is perplexed about what she&rsquo;s looking at. Cube tries to explain: &#10024; Theorizing achievement unlocked! &#10024; Whatever this structure is, it&rsquo;s too large for Sphere or Cube (or even reader) to see the whole thing at once. A theory is needed, and theories need reasons. The bottom half is right-angled and pointy like Cubes, and the top half is spherical like Spheres. Given the enormity of the structure, and the unlikely blending of what they&rsquo;ve both only known as two distinct shapes, Cube shares his theory for what this structure mean: despite their differences, they come from the same place. As cognitive shapes in the universe, they are more alike than different. Cube argues that this is the reason that Cube trusts Sphere, and has come to her for help, and why Sphere should trust Cube enough to help him. Cube invites Sphere to have this conversation, now that his full argument has been heard. Sphere accepts this invitation, re-opens the old associations that address Sphere+Cube relations, and they re-ignite as &#128293; as they were on the first day. We&rsquo;ve come a long way from the first &ldquo;Huh?&rdquo; and the delayed, but eventual &ldquo;Oh.&rdquo; on that steep cliff at the beginning of the universe. Consider also how far we&rsquo;ve come from the conversation on Sphere&rsquo;s door at the beginning of this level. The number of possible ways to traverse the distance between &ldquo;Huh?&rdquo; and &ldquo;Oh.&rdquo; has greatly increased, but the basic premise has remained the same. This is some advanced communicating we&rsquo;re seeing now. Yay team. Let&rsquo;s review some&nbsp;things We&rsquo;ve watched Sphere (and Cube!) evolve capacity for emotions, language, self-defense, and the theory of mind that others also have independent agency and motivations. Though mistakes have been made, the evolution of reason has pulled through for Sphere and Cube. It has become their go-to tool for better predicting how others reason and behave, which helps us adjust how we reason and behave, which helps them adjust, etc. Everyone wins. However, there&rsquo;s a conversation happening below the threshold of our direct attention that we unfortunately can&rsquo;t afford to ignore, and this old, grumpy conversation will only ever operate in the original language of brute force: Natural selection insists that the fittest shall&nbsp;survive. Sphere and Cube can continue to build better tools for everyday use, but they still have to work within this larger context. This presents them with a quandary. The most reasonable, and the smartest, are not guaranteed to survive. Not even the fairest bargainer. What does this mean for reason? Who does reason ultimately answer&nbsp;to? Reason answers to Truth and Growth. Above all, it favors accuracy (useful when making predictions about our mind and other minds), seeks correction, and has no hidden agenda. It&rsquo;s a spiritual descendant of core values like honesty, integrity, and courage. Reason answers to Life and Winning. Above all, reasoning is a tool for improving chances of survival, winning arguments and making excuses to cover your butt when things go wrong. In this way it&rsquo;s a more clever sibling to brute force and bargaining, but part of the same family none-the-less. It doesn&rsquo;t seek accuracy, unless accuracy helps chances of survival. It doesn&rsquo;t seek correction unless seeking correction helps chances of survival. It&rsquo;s a spiritual descendent of wartime accessories like the sword, the shield, and the helmet. These are two very different things to possibly answer to. Sphere and Cube are in a difficult moment right now. The quiet, but incredibly stubborn, conversation about survival has raised its voice. Is Sphere&rsquo;s reason for withholding help from Cube still good enough? Does Cube&rsquo;s explanation about the reason behind their desperation change anything in Sphere&rsquo;s story about them? When survival is at risk on both sides, what will reason ultimately answer to? &#10024; Opaque Reasoning achievement unlocked! &#10024; If we asked Sphere why she agreed to help Cube, she might not even know. We can be sure that she didn&rsquo;t necessarily weigh every piece of evidence, consider every option, and compute a logical deduction of facts, risks, and opportunities. There&rsquo;s not enough time remaining in the universe to do that. So what did she do, if not reason it out? Is her decision rational? And does it provide us any clues about whether reason answers to Truth or Life? We&rsquo;ll come back to this, but first let&rsquo;s see how this story ends. &#10024; Courage achievement unlocked! &#10024; &#127942; Level 2&#65039;&#8419; complete! &#128079; &#128079; &#128079; &#128275; Level 3&#65039;&#8419; unlocked! &#128079; &#128079; &#128079; Summing up: what is reasoning for? The simplest way to define reasoning is like this: It&rsquo;s what gets you from &ldquo;Huh?&rdquo; to &ldquo;Oh.&rdquo; as often as possible, for as broad a variety of &ldquo;Huh?&rdquo;s out&nbsp;there. Here is an incomplete list of different strategies we&rsquo;ve evolved, all starting back with the very first conscious thought: Sometimes just opening your eyes and looking at what you just bumped into is enough to get there. &ldquo;Oh, I walked right into a pole.&rdquo; That&rsquo;s called Empirical Reasoning, and extends to include the full set of things that you can figure out just by looking at things in various ways. Sometimes you see something and it&rsquo;s just not right. So you move it, push it, or do something else to it to make it right. And you say, &ldquo;Oh, that&rsquo;s much better.&rdquo; Let&rsquo;s call this Brute Force Reasoning. Sometimes a &ldquo;huh?&rdquo; exists within a social context that enforces some rules around what&rsquo;s considered &ldquo;fair&rdquo; (though what&rsquo;s considered fair may or may not actually be fair). In these cases, a deal needs to be made. For example, someone wants to you to work for them, and they&rsquo;re willing to pay you to do it. Let&rsquo;s call this Bargain Reasoning. When Cube asked Sphere for help, none of the above types of reasoning would have been effective. In the end, Sphere helped Cube for a number of reasons, some conscious and some probably unconscious: She saw how Cubes and Spheres could be related. She received information about them (they were dying of cold) that challenged her own story about them being pure monsters. Perhaps she realized that by giving them fire voluntarily, they might stop using Brute Force Reasoning, and they would stop being terrible. This mixed bag of &ldquo;reasons&rdquo;, in addition to Sphere&rsquo;s mood at the time, the amount of sleep she got last night, and the position of the stars in the sky, is what we traditionally call Logical Reasoning. But that&rsquo;s probably not the right name for it&hellip; because in reality it&rsquo;s far from logical. In order to maintain that name, the field of psychology, economics, and many other sciences have been diligent about identifying bugs in this reasoning: cognitive biases, logical fallacies, implicit associations, etc. But over the last decade or so an alternative interpretation has slowly been gaining steam, which I think makes a lot more sense. Perhaps our logic isn&rsquo;t flawed, it&rsquo;s that we&rsquo;ve called it the wrong thing! If you pick up a rock, but call it a duck, and then start finding a million reasons why it&rsquo;s a pretty terrible duck, it might make sense to at some point reconsider its duckness! Let&rsquo;s call it a rock instead, and then we&rsquo;ll see how it&rsquo;s actually a pretty great rock. If we change the name to Motivated Reasoning then all of the bugs become features. Sphere, you see, is motivated to live. She wants Cube to stop killing her. The giant symbol they found may or may not be what Cube says it is, but if they both agreed to give it the meaning of &ldquo;we&rsquo;re from the same shape&rdquo; that would increase her chances of survival. And therefore fulfill the motivation for this reason to be accepted. Building reason-based and experience-based trust between them would also increase her chances of survival. Giving them fire voluntarily, while still alive, would obviously increase her chances as well. In the clearest words possible: Being illogical in this situation was the most reasonable thing for her to do, considering her motivations! And so, perhaps, we have our answer. Reason answers first to Life, Winning, and most importantly, Survival. It has helped us over millions of years to compete for survival, and win. To get from &ldquo;Huh?&rdquo; to &ldquo;Oh.&rdquo; in the most diverse set of circumstances, we must actually be alive for that &ldquo;Oh.&rdquo; to happen! ba-dum tss The open&nbsp;question If we accept that logic, rationality, and objective truth will never be the primary objective of reasoning, does that mean we should give up our ambitions in those areas? Probably not. So what&rsquo;s the path forward? Here&rsquo;s what I&rsquo;m wondering right now. How might we embrace our survival ultimatum from natural selection and use it as inspiration for having more honest, heated conversations that make us smarter and bring us closer together? For example: In the context of natural selection, you couldn&rsquo;t say that today&rsquo;s humans are the best and truest humans that will ever be, and be absolutely certain about that. Evolution isn&rsquo;t ever &ldquo;done&rdquo;. All you can really say is that we&rsquo;ve been the most successful at surviving, so far, within the ecological niches that we&rsquo;ve attempted to be competitive within. (And honestly hope that we are good enough to survive the next few rounds, and still have an opportunity to grow.) SIMILARLY, in the context of our beliefs about life, politics, work, relationships, etc, we can&rsquo;t say that any one of our current arguments is the best and truest argument that will ever be, and be absolutely certain about it. All we can hope for is that it&rsquo;s been the most successful at surviving, so far, within the heated discussions that it has participated in. And because reasons and arguments do impact our chances for survival (both literally and more short term within the realm of social reputation) we should not hold on to arguments that aren&rsquo;t likely to survive future heated discussions. It&rsquo;s evolutionarily advantageous to us to excitedly toss less fit arguments aside as soon as more fit arguments are discovered! That part of Motivated Reasoning that wants to be right in this moment in order to save face just needs to be nudged to consider that being right for the rest of our lives (after we abandon this argument for the better one) is a much better evolutionary strategy. It can keep the same end goal, just shift the framing a tad. And since the fitness of arguments can only really be tested in heated discussions and debates we should be seeking out as many of these as we can&#8202;&mdash;&#8202;not in the spirit of winning, but in the spirit of finding better arguments to put in our Pokedex, I mean Beliefdex. And, obviously, we should be seeking out the best opposing arguments rather than the worst, because the best ones are candidates for adoption, not just target practice (aka steelmen instead of strawmen). And, if we&rsquo;re really serious, we should invite people from as diverse a range as possible to construct stronger arguments together, arguments designed to crush our current best argument on any given topic. So. Who&rsquo;s in? In the spirit of collecting as diverse a selection of arguments to work with, can you share a heated argument that you&rsquo;ve had in the past, or are currently having? &#128073; Share an example of a heated argument you&rsquo;ve&nbsp;had Thank you! &#128591; Acknowledgments I&rsquo;m obviously not the first person to become obsessed with the topic of reason and heated conversations. In my early research, I&rsquo;ve found a number of people who have been thinking about this for a long time already: Julia Galef and her Rationally Speaking podcast has been one of my favorite sources of new ideas, like steelmanning. And the rabbit hole has lead to other interesting things like the disagreement hierarchy (D07 requires you to fix any obvious flaws in your opponent&rsquo;s argument before refuting it), enumerating causes of disagreements, and more. The argumentative theory of reasoning is also super fascinating and worth reading. &#127873; Bonus prize for getting to the end Monty Python sketch about an argument clinic where you pay people to argue with you (the first 3 minutes are best). Show your support Clapping shows how much you appreciated Buster Benson&rsquo;s story. Thinking Is Hard Where I write about writing a book about biases, reasoning, and having better heated discussions.<br>
A minimalist comic about the evolution of reasoning and heated conversations<br>
Label: grist<br>
Date: August 27, 2017 at 10:22AM
